Mathlib/Tactic/Find.lean

```diff
@@ -43,9 +43,9 @@ private partial def matchHyps : List Expr → List Expr → List Expr → MetaM
 -- from Lean.Server.Completion
 private def isBlackListed (declName : Name) : MetaM Bool := do
   let env ← getEnv
-  declName.isInternal
-  <||> isAuxRecursor env declName
-  <||> isNoConfusion env declName
+  pure $ declName.isInternal
+   || isAuxRecursor env declName
+   || isNoConfusion env declName
   <||> isRec declName
```

Comment:
It surprised me that this worked without parenthesizes.

Reply:
Yeah, the precedence is not what I would have expected. See also the `!` precedence below.